.github/workflows/snyk-sbom.yml

```diff
@@ -0,0 +1,25 @@
+name: Snyk - Software Bill of Materials (SBOM)
+on:
+  release:
+    types: [published]
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - name: Generate SBOM
+        uses: snyk/actions/node@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          command: sbom
+          args: --format=cyclonedx1.5+json --all-projects > sbom.json
```

Comment:
Should we also attache the release version to the file name ? 
Something like `mattermost-9.5.2-sbom.json` . |
Also should we sign the file ?

Reply:
@toninis I don't think we need to rename it since it will be listed under that specific release and I don't see our other release artifacts being named. I can add this later though. I would prefer to get it released and rename the file if necessary at a later date.

I have the same thought on signing the file unless @esarafianou disagrees.

Reply:
Let's start with the current PR and iterate on improving it in a followup one. Signing would be valuable indeed, but we can add it later.